sda-commons-server-kafka/src/main/java/org/sdase/commons/server/kafka/topicana/TopicComparer.java

```diff
@@ -36,7 +33,7 @@
 import org.sdase.commons.server.kafka.KafkaProperties;
 
 /**
- * This is a copy of the original {@link com.github.ftrossbach.club_topicana.core.TopicComparer}
```

Comment:
Don't we need to add the license to this file too?

Reply:
Look at the start of this file :)

Reply:
We already have one, perfect